source/image-handler/image-handler.ts

```diff
@@ -19,6 +20,11 @@ export class ImageHandler {
    */
   async process(imageRequestInfo: ImageRequestInfo): Promise<string> {
     const { originalImage, edits } = imageRequestInfo;
+    const answer = isAnimated(originalImage) ? 'Yes' : 'No';
+
+    if (answer === 'Yes') {
```

Comment:
Since the PR is intended to support `gif` a returning the image in L26 leads to that some filters supported by sharp library, like `crop` won't be applied.

Reply:
This was a work in progress sorry, didn't want to make pull request to your repo, it was intended to do it in the fork

Reply:
I understand, no worries. Than you for your contribution.